ui/radio.mod/radio.js

```diff
@@ -1,95 +1,79 @@
 /**
-    @module "mod/ui/input-radio.mod"
-*/
-var CheckControl = require("ui/check-control").CheckControl,
-    PressComposer = require("../../composer/press-composer").PressComposer,
-    KeyComposer = require("../../composer/key-composer").KeyComposer;
+ *   @module "mod/ui/input-radio.mod"
+ */
+const { KeyComposer } = require("../../composer/key-composer");
+const { CheckControl } = require("ui/check-control");
+
 /**
  * Wraps the a &lt;input type="radio"> element with binding support for the element's standard attributes.
-   @class module:"mod/ui/native/input-radio.mod".InputRadio
-   @extends module:mod/ui/check-input.CheckInput
+ * @class module:"mod/ui/native/input-radio.mod".InputRadio
+ * @extends module:mod/ui/check-input.CheckInput
  */
-exports.Radio = CheckControl.specialize({
-
-    // constructor: {
-    //     value: function InputRadio() {
-    //         this.super();
-    //         return this;
-    //     }
-    // },
-
-    drawsFocusOnPointerActivation: {
-        value: true
-    },
-
-    hasTemplate: {
-        value: false
-    },
-
-    _keyComposer: {
-        value: null
-    },
-
-    _radioButtonController: {
-        value: null
-    },
-
-    enterDocument: {
-        value: function (firstTime) {
-            if (firstTime) {
-                this.element.setAttribute("role", "radio");
-            }
-        }
-    },
-
-    prepareForActivationEvents: {
-        value: function() {
+exports.Radio = class Radio extends CheckControl {
+    // <---- Properties ---->
 
-            this.super();
+    drawsFocusOnPointerActivation = true;
 
-            this._keyComposer = new KeyComposer();
-            this._keyComposer.component = this;
-            this._keyComposer.keys = "space";
-            this.addComposer(this._keyComposer);
+    _radioButtonController = null;
```

Comment:
put in static block with 
static {
      Montage.defineProperties(this.prototype, {...}
}